Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml

```diff
@@ -561,188 +561,218 @@ script:
       contextPath: CiscoESA.Message.sbrs
       description: Sender Base Reputation Scores.
   - name: cisco-esa-message-details-get
-    description: Get message details.
+    description: Retrieve metadata and tracking information for email messages processed by the ESA. Includes sender/recipient details, timestamps, message status (e.g., delivered, dropped), message size, AMP and DLP visibility flags, and applied mail policies — useful for auditing and threat tracing.
```

Comment:
Retrieves metadata and tracking information for email messages processed by the ESA. Includes sender/recipient details, timestamps, message status for example, delivered, dropped), message size, AMP and DLP visibility flags, and applied mail policies. Useful for auditing and threat tracking.